### DISCUSSION TITLE
Are there some DolphinScheduler Benchmarking Tools？

### DISCUSSION THREAD

**Discussion by lawlielt** [apache/dolphinscheduler#14567] [Q&A]
like this [performance-test-tool](https://medium.com/@ApacheDolphinScheduler/performance-test-the-apache-dolphinscheduler-schedules-twice-as-many-concurrent-tasks-per-minute-32d1ee0131b0)

**Comment by ruanwenjun** [comment]
Right now there is no benchmark tools, but there is a plan to create a module for this.
